ARCs/arc-0028.md

```diff
@@ -0,0 +1,81 @@
+---
+arc: 28
+title: Algorand Event Log Spec
+status: Draft
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/113
+---
+
+# Algorand Event Log Spec
+
+> This API is a draft.
+> Some elements may change.
+
+## Summary
+
+A methodology for structured logging by Algorand dapps.
+
+## Abstract
+
+Algorand dapps can use the `log` primitive to attach information about an application call. This ARC proposes the concept of Events, which are merely a way in which data contained in these logs may be categorized and structured.
```

Comment:
As a suggestion rather than "dapps" should this say smart contracts to make it clearer (on first read I thought it was talking about `console.log` in a web app, which is normally what I think when hearing "dapp").

Also suggest linking `log` to <https://developer.algorand.org/docs/get-details/dapps/avm/teal/opcodes/#log>.